crates/examples/src/bin/readobj.rs

```diff
@@ -82,6 +82,12 @@ fn main() {
                 .action(ArgAction::SetTrue)
                 .help("Print the Mach-O load commands"),
         )
+        .arg(
+            Arg::new("macho-function-starts")
+                .long("macho-function-starts")
+                .action(ArgAction::SetTrue)
+                .help("Print the Mach-O function start addresses"),
+        )
```

Comment:
I don't think there's a need to add a new option for this. `macho-load-commands` is sufficient.

Reply:
happy to make the change, but I thought a new flag was reasonable since 1) `readobj` has a variety of `--elf-*` flags already, 2) the text dump of the function starts contents could be 100k+ lines for moderately sized programs, much larger than the existing load commands dump size and 3) I was thinking of implementing dumping of the exports trie and bind opcodes next, which would also result in rather long outputs.